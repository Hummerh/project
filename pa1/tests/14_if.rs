if b'c' == b'\'' {
      b"Hello world"
}
