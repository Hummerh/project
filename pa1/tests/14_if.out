if
LITCHAR(c)
==
LITCHAR(')
{
LITSTR(Hello world)
}
